include/snowflake/SF_CRTFunctionSafe.h

```diff
@@ -26,40 +26,6 @@ extern "C" {
     #endif
 #endif
 
-// Defined for #pragma warning messages.
-#define MACRO_TO_STRING2(x) #x
-#define MACRO_TO_STRING(x) MACRO_TO_STRING2(x)
-
-#if defined(_MSC_VER)
-#define SF_MACRO_DEPRECATED_WARNING(MSG) __pragma(message ( __FILE__ "(" MACRO_TO_STRING(__LINE__) ") : warning C4996: " MSG))
```

Comment:
Please move the macro to a different place - I think there will be more places we would like to mark in the future

Reply:
Makes sense. Though I don't see currently a proper header to keep these macros - shall we create new `common.h`?

Reply:
common.h could be created or we can add it to include/snowflake/version.h - deprecating is related to the versioning

Reply:
platform.h is fine I think.